modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/270_small_retention.yml

```
@@ -0,0 +1,53 @@
+     indices.delete_data_stream:
+        name: "tiny-retention-data-stream"
+        ignore: 404
+  - do:
+     indices.delete_index_template:
+        name: "tiny-retention-template"
+        ignore: 404
+
+---
+"Test failure store rollover with tiny retention":
+  - requires:
+      test_runner_features: [ capabilities, allowed_warnings ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)